ci-operator/config/openshift/release/openshift-release-master__okd-scos-4.15.yaml

```
@@ -3,8 +3,9 @@ releases:
     prerelease:
       product: okd
       version_bounds:
-        lower: 4.12.0
-        upper: 4.1000.0
+        lower: 4.13.0-0
```

```suggestion
        lower: 4.14.0-0
```

Not sure if this picks 4.14 without this change

---

see https://github.com/openshift/release/pull/42843#discussion_r1311260404